client/core/core.go

```diff
@@ -3790,6 +3790,9 @@ func handleRevokeOrderMsg(c *Core, dc *dexConnection, msg *msgjson.Message) erro
 
 	tracker.revoke()
 
+	details := fmt.Sprintf("Order %s on market %s at %s has been revoked by the server", tracker.token(), tracker.mktID, dc.acct.host)
+	c.notify(newOrderNote("Order Revoked", details, db.ErrorLevel, tracker.coreOrderInternal()))
```

Comment:
This needs to be `coreOrder()` instead of `coreOrderInternal()` since the trade's mutex is not locked at this point.